### PR TITLE
[not wanted] Makes meteor unvotable

### PR DESCRIPTION
**Pull request by DreamySkrell**
It is a meme gamemode and should not exist.

Meteor does not offer any gameplay for anyone but engineering, and I believe it is wholly detrimental to the HRP that we should aim to provide.

Anyone voting for it is trolling.

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

File: html/changelogs/DreamySkrell-meteor-unvotable.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - tweak: "Makes meteor gamemode unvotable."
```
Comment (Generalcamo) [inline html/changelogs/DreamySkrell-meteor-unvotable.yml:41]:
```suggestion
  - rscdel: "Makes meteor gamemode unvotable."
```

**Review by Generalcamo — CHANGES_REQUESTED** [review]

File: html/changelogs/DreamySkrell-meteor-unvotable.yml (hunk 1 above)
Comment (DreamySkrell) [reply html/changelogs/DreamySkrell-meteor-unvotable.yml:41]: It is not removal, and meteor still exists.

**Comment by meep109** [comment]
I would like a feedback thread opened on this

**Comment by DreamySkrell** [comment]
https://forums.aurorastation.org/topic/18827-meteor-removal-requestfeedback/

**Pull request closed without merging**